core/driver/sources/src/test/resources/example.conf

```diff
@@ -2,7 +2,7 @@ probe {
   resolvers {
     intellij {
       repositories = [
-        "https://www.jetbrains.com/intellij-repository/snapshots/[orgPath]/[module]/[artifact]/[revision]/[artifact]-[revision].zip"
+        "https://www.jetbrains.com/intellij-repository/snapshots/com/jetbrains/intellij/idea/ideaIC/[revision]/ideaIC-[revision][format]"
```

Comment:
the test was checking if the `orgPath` `module` and `artifact` are set correctlly, why inline those?

Reply:
I changed it as [orgPath], [module] and [artifact] were always resolved to the same constant string values. Should I revert this change? Add these replacaments back to the scala code?

Reply:
yeah, +1, revert (it might be sth else than `ideaIC` indeed, even just `ieadIU`)

Reply:
yeah, for example iu/ic is controlled by resolvers, but we might add `type/product` key to config and pick specific variant.